src/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import reducer from './reducer';
+
+const rootReducer = combineReducers({
+  count: reducer,
```

Comment:
the name `reducer` is not good. `count` or ` counter` is better.
you should change the file name `reducer.js` to `counter.js`